--- .rcairo-1.16.5/ext/cairo/extconf.rb.orig	2020-02-05 17:45:41.611038949 +0100
+++ .rcairo-1.16.5/ext/cairo/extconf.rb	2020-02-05 17:47:02.103936215 +0100
@@ -33,30 +33,7 @@
   end
 end
 
-def required_pkg_config_package(package_info, native_package_info=nil)
-  if package_info.is_a?(Array)
-    required_package_info = package_info
-  else
-    required_package_info = [package_info]
-  end
-  return true if PKGConfig.have_package(*required_package_info)
-
-  native_package_info ||= {}
-  return false unless NativePackageInstaller.install(native_package_info)
-
-  PKGConfig.have_package(*required_package_info)
-end
-
-unless required_pkg_config_package([package, major, minor, micro],
-                                   :arch_linux => "cairo",
-                                   :debian => "libcairo2-dev",
-                                   :homebrew => "cairo",
-                                   :macports => "cairo",
-                                   :msys2 => "cairo",
-                                   :redhat => "cairo-devel")
-  exit(false)
-end
-
+PKGConfig.have_package(package, major, minor, micro)
 PKGConfig.have_package("cairo-ft")
 
 checking_for(checking_message("macOS")) do
