diff -ur rcairo-1.15.10/ext/cairo/extconf.rb rcairo-1.15.10-no-native-pkgs/ext/cairo/extconf.rb
--- rcairo-1.15.10/ext/cairo/extconf.rb	2017-10-15 16:21:01.000000000 +0200
+++ rcairo-1.15.10-no-native-pkgs/ext/cairo/extconf.rb	2017-11-09 09:19:45.074637717 +0100
@@ -43,29 +42,7 @@
   end
 end
 
-def required_pkg_config_package(package_info, native_package_info=nil)
-  if package_info.is_a?(Array)
-    required_package_info = package_info
-  else
-    required_package_info = [package_info]
-  end
-  return true if PKGConfig.have_package(*required_package_info)
-
-  native_package_info ||= {}
-  return false unless NativePackageInstaller.install(native_package_info)
-
-  PKGConfig.have_package(*required_package_info)
-end
-
-unless required_pkg_config_package([package, major, minor, micro],
-                                   :debian => "libcairo2-dev",
-                                   :redhat => "cairo-devel",
-                                   :homebrew => "cairo",
-                                   :macports => "cairo",
-                                   :msys2 => "cairo")
-  exit(false)
-end
-
+PKGConfig.have_package(package, major, minor, micro)
 PKGConfig.have_package("cairo-ft")
 
 checking_for(checking_message("Mac OS X")) do
